style.css

```diff
@@ -20,6 +21,22 @@ a {
   text-decoration: none;
 }
 
+#contents > a > section {
```

Comment:
I'm curios to understand why you did not specified the `:nth-child` here. The current code makes this line of code to be always overwritten by the below rules.